src/getHardhatConfigNetworks/index.test.ts

```
@@ -0,0 +1,174 @@
+        },
+        hardhat: {
+          chainId: 1337,
+          forking: { blockNumber: 14672712, url: "https://rpc.ankr.com/eth" }
+        }
+      };
+    }
+  };
+});
+
+import * as getHardhatConfigNetworksModule from './index';
+import { getHardhatConfigNetworks as actualGetHardhatConfigNetworks } from './getHardhatConfigNetworks';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)